notebooks/test_notebooks.py

```diff
@@ -270,8 +270,9 @@ def test_responsibleaidashboard_fridge_image_classification_model_debugging():
 @pytest.mark.vision_notebooks
 def test_responsibleaidashboard_fridge_multilabel_ic_model_debugging():
     nb_path = RESPONSIBLEAIDASHBOARD
-    nb_name = "responsibleaidashboard-fridge-multilabel-" + \
-              "image-classification-model-debugging"
+    nb_name = \
+        "responsibleaidashboard-fridge-multilabel-" + \
```

Comment:
nit: this might be my personal preference so feel free to ignore but I like to use parens instead of \ slashes